projects/1.18/assets/untamedwilds/untamedwilds/lang/zh_cn.json

```
@@ -0,0 +1,726 @@
+  "block.untamedwilds.carpet_bear_black": "黑色熊皮地毯",
+  "block.untamedwilds.carpet_bear_brown": "棕色熊皮地毯",
+  "block.untamedwilds.carpet_bear_white": "白色熊皮地毯",
+  "block.untamedwilds.carpet_bigcat_jaguar": "美洲豹皮地毯",
+  "block.untamedwilds.carpet_bigcat_leopard": "花豹皮地毯",
+  "block.untamedwilds.carpet_bigcat_lion": "狮子皮地毯",
+  "block.untamedwilds.carpet_bigcat_panther": "黑豹皮地毯",
+  "block.untamedwilds.carpet_bigcat_puma": "美洲狮皮地毯",
+  "block.untamedwilds.carpet_bigcat_snow_leopard": "雪豹皮地毯",
+  "block.untamedwilds.carpet_bigcat_tiger": "虎皮地毯",
+  "block.untamedwilds.trap_cage": "捕捉笼",
+  "block.trap_cage.state_empty": "空",
+  "block.trap_cage.state_full": "包含：",
+  "block.untamedwilds.block_burrow": "地洞",
+  "block.burrow.state": "这个地洞属于某种%s",
+  "block.untamedwilds.nest_reptile": "爬行动物巢穴",
+  "block.untamedwilds.nest_amphibian": "两栖动物巢穴",
+  "block.untamedwilds.feeder": "吊挂式饲槽",
+
+  "advancements.untamedwilds.zoologist.title": "动物",
+  "advancements.untamedwilds.zoologist.description": "是时候去探索世界，制作生活百科全书了",
+  "advancements.untamedwilds.taming_bear.title": "熊军一号",
+  "advancements.untamedwilds.taming_bear.description": "驯服一只熊",
+  "advancements.untamedwilds.basic_baiter.title": "老兄",
+  "advancements.untamedwilds.basic_baiter.description": "在水中用一块鱼食吸引附近的水生生物",
+  "advancements.untamedwilds.master_baiter.title": "饵食大师",
+  "advancements.untamedwilds.master_baiter.description": "用一块鱼食吸引一百多个生物",
+  "advancements.untamedwilds.clam_digger.title": "挖砗人",
+  "advancements.untamedwilds.clam_digger.description": "用铲子挖出一只巨大的砗磲",
+  "advancements.untamedwilds.hog_wild.title": "发疯",
+  "advancements.untamedwilds.hog_wild.description": "试着骑一头野猪（但是失败了）",
+  "advancements.untamedwilds.yee_haw.title": "驾",
+  "advancements.untamedwilds.yee_haw.description": "参加世界上最糟糕的牛仔表演",
+  "advancements.untamedwilds.discovered_spitter.title": "为什么耷拉着脸？",
+  "advancements.untamedwilds.discovered_spitter.description": "发现一只喷吐者",
+  "advancements.untamedwilds.activated_feeder.title": "捕虫手",
+  "advancements.untamedwilds.activated_feeder.description": "在喂食器上放一些蜂蜜以吸引昆虫",
+  "advancements.untamedwilds.lepidopterologist.title": "蝴蝶学家",
+  "advancements.untamedwilds.lepidopterologist.description": "捉住每一种蝴蝶",
+
+  "tetra.variant.sword_socket/pearl": "槽位 [§3Pearl§r]",
+  "tetra.variant.sword_socket/pristine_pearl": "槽位 [§3Pearl+§r]",
+  "tetra.variant.sword_socket/pristine_pearl.description": "一颗宝石镶嵌在剑刃与剑柄相接的位置。宝石已经附魔，为所有者提供魔法加持。\n\n这颗宝石是原始品质，提供比低等宝石更好的效果。",
+  "tetra.variant.double_socket/pearl": "槽位 [§3Pearl§r]",
+  "tetra.variant.double_socket/pristine_pearl": "槽位 [§3Pearl+§r]",
+  "tetra.variant.double_socket/pristine_pearl.description": "一颗宝石镶嵌在剑刃与剑柄相接的位置。宝石已经附魔，为所有者提供魔法加持。\n\n这颗宝石是原始品质，提供比低等宝石更好的效果。",
+  "tetra.variant.single_socket/pearl": "槽位 [§3Pearl§r]",
+  "tetra.variant.single_socket/pristine_pearl": "槽位 [§3Pearl+§r]",
+  "tetra.variant.single_socket/pristine_pearl.description": "一颗宝石镶嵌在剑刃与剑柄相接的位置。宝石已经附魔，为所有者提供魔法加持。\n\n这颗宝石是原始品质，提供比低等宝石更好的效果。"

```

```suggestion
  "tetra.variant.sword_socket/pearl": "镶嵌[§3珍珠§r]",
  "tetra.variant.sword_socket/pristine_pearl": "镶嵌[§3珍珠+§r]",
  "tetra.variant.sword_socket/pristine_pearl.description": "一颗宝石，被装进剑刃与剑柄的连接处。这颗宝石附有魔咒，会为持有者提供魔法增益\n \n这颗宝石质地无瑕，能提供更好的效果",
  "tetra.variant.double_socket/pearl": "镶嵌[§3珍珠§r]",
  "tetra.variant.double_socket/pristine_pearl": "镶嵌[§3珍珠+§r]",
  "tetra.variant.double_socket/pristine_pearl.description": "一颗宝石，被装进工具的顶端。这颗宝石附有魔咒，会为持有者提供魔法增益\n \n这颗宝石质地无瑕，能提供更好的效果",
  "tetra.variant.single_socket/pearl": "镶嵌[§3珍珠§r]",
  "tetra.variant.single_socket/pristine_pearl": "镶嵌[§3珍珠+§r]",
  "tetra.variant.single_socket/pristine_pearl.description": "一颗宝石，被装进头部与手柄的连接处。这颗宝石附有魔咒，会为持有者提供魔法增益\n \n这颗宝石质地无瑕，能提供更好的效果"
```
Tetra兼容，谢谢审核
#2243 #3179 #3328 #3148
多谢

---

还有一个#2815 差点忘了

---

谢谢！tetra我实在不太熟悉，感谢捉虫